backend/people/migrations/0005_fill_holding_team_data.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
+def forwards(apps, schema_editor):
+    Staff = apps.get_model('people', 'Staff')
+    teams = apps.get_model('people', 'HoldingTeam')
+    team = teams.objects.create(
+        name='content'
+    )
+    team.staff.add(
+        Staff.objects.filter(name__in=['Mehdi Farvardin', 'Hossein Firooz', 'Sepehr Amini Afshar', 'Farzam Zohdinasab', 'Pooya Moeini', 'Seyed Mohammad mehdi Hatami'])
+    )
+    team = teams.objects.create(
+        name='technical'
+    )
+    team.staff.add(
+        Staff.objects.filter(name__in=['Emran Batman ghelich', 'Ahmad Salimi', 'Ali asghar Ghanati', 'Fateme Khashei', 'Alireza Tajmir riahi', 'Mohammad mehdi Barghi', 'Seyed Alireza Hashemi', 'ArhsiA Akhavan'])
+    )
+    team = teams.objects.create(
+        name='network'
+    )
+    team.staff.add(
+        Staff.objects.filter(name__in=['Amirhossein Hadian', 'Amirmohammad Imani', 'Sajjad Rezvani', 'Shima Ramadani', 'Mehdi Jalali', 'Sara Azarnoosh', 'Ehsan Movafagh'])
+    )
+    team = teams.objects.create(
+        name='branding'
+    )
+    team.staff.add(
+        Staff.objects.get(name='Seyed Alireza Hosseini')
+    )
+    team = teams.objects.create(
+        name='social'
+    )
+    team.staff.add(
+        Staff.objects.filter(name__in=['Sara Azarnoosh', 'Dorna Dehghani', 'Ghazal Shenavar', 'Helia Akhtarkavian', 'Sabiheh Tajdari', 'Sahel Messforoosh', 'Esmaeil Pahang'])
+    )
+    team = teams.objects.create(
+        name='media'
+    )
+    team.staff.add(
+        Staff.objects.filter(name__in=['Hamila Meili', 'Mahdieh Ebrahimpoor', 'Roya Aghvami', 'Sara Zahedi', 'Hossein Aghamohammadi'])
+    )
+    team = teams.objects.create(
+        name='presentation management'
+    )
+    team.staff.add(
+        Staff.objects.filter(name__in=['Alireza Ziaei', 'Amirhossein Asem Yousefi', 'Vahid Zehtab', 'Sajjad Rezvani'])
+    )
+
+
+def rollback(apps, schema_editor):
+    teams.objects.get(name='content').delete()
```

Comment:
This may remove data from last year. Try this:
```Python
wss = WSS.objects.get(year=2020)
teams.objects.filter(wss=wss, name__in=names).delete()
```